tests/unit/checks/test_check_crl.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019 tribe29 GmbH - License: GNU General Public License v2
+# This file is part of Checkmk (https://checkmk.com). It is subject to the terms and
+# conditions defined in the file COPYING, which is part of this source code package.
+
+import pytest  # type: ignore[import]
+from testlib import ActiveCheck  # type: ignore[import]
+
+pytestmark = pytest.mark.checks
+
+
+@pytest.mark.parametrize("params,expected_args", [
+    (("foo", 222, 111, "bar", {}), ["--url=foo", "--warn=222", "--crit=111"]),
```

Comment:
According to the valuespec, you expect a dict here, not a tuple. Accordingly, this test failed in the last travis run.